### DISCUSSION TITLE
lost my connection to Database after running VPN

### DISCUSSION THREAD

**Discussion by HazemAlShibani** [prisma/prisma#24947] [Q&A]
### Question

hey, 
i faced this error "**_Can't reach database server at `ep-cool-hill-a2vsznaj.eu-central-1.aws.neon.tech:5432`_**" when I working on website using Next.js and the problem is happened when I run VPN to necessary reason, I lost the connection at all, but when I stop VPN everything comeback fine...
is there any way to use Prisma database in this situation cause I need to use VPN in my application!!?

**Comment by ludralph** [comment]
Hi @HazemAlShibani 👋 

Can you connect with any other tool using that connection string when connected to your VPN? (e.g. DBeaver or some other database GUI)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
